doc/pandoc.txt

```diff
@@ -236,7 +236,7 @@ Besides this, some extensions are provided:
 * If present, fold the document's YAML frontmatter (this is configurable with
   |g:pandoc#folding#fold_yaml|)
 * The foldlevel for YAML frontmatter is 1 by default (this is configurable
-  with |g:pandoc#folding#folding_yaml|)
+  with |g:pandoc#folding#foldlevel_yaml|)
```

Comment:
It looks like you are correct that there is a problem here but incorrect about the solution. There *is* a `g:pandoc#folding#fold_yaml` setting being set and used in  the code, I don't see where your `...#foldlevel_yaml` came from.

Reply:
`fold_yaml` is to enable yaml folding. `foldlevel_yaml` is to configure the fold level for the yaml frontmatter. Useful in cases where I want all other folds to open by default except yaml frontmatter.

Weirdly search doesn't show all instances of `foldlevel_yaml`, [here](https://github.com/vim-pandoc/vim-pandoc/commit/9be2bbeb826ce8b7827ee37326ac90e221c83736) is when this feature was introduced.

Reply:
Thanks for the correction, I see now what was going on. As for my local search, it was flawed because somehow I was poking at a really old clone that hadn't been updated since the feature was added.